business_requirement/views/business_view.xml

```diff
@@ -251,7 +251,7 @@
         <field name="name">Business Requirement</field>
         <field name="res_model">business.requirement</field>
         <field name="view_mode">tree,kanban,form,graph,pivot</field>
-        <field name="view_type">form</field>
+        <field name="binding_view_types">form</field>
```

Comment:
You must remove this, as this is for binding an action in the dropdown menu "Actions".